src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Shape {
+    private double top;
+    private double bottom;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double top, double bottom, double height) {
+        super(Colors.valueOf(color));
+        this.top = Math.round(Math.abs(top));
+        this.bottom = Math.round(Math.abs(bottom));
+        this.height = Math.round(Math.abs(height));
+    }
+
+    public double getTop() {
+        return top;
+    }
+
+    public void setTop(double top) {
+        this.top = top;
+    }
+
+    public double getBottom() {
+        return bottom;
+    }
+
+    public void setBottom(double bottom) {
+        this.bottom = bottom;
```

Comment:
Consider validating the `bottom` value in the `setBottom` method to ensure it is non-negative, similar to the constructor.